packages/trader/src/Modules/Trading/Components/Form/TradeParams/barrier.jsx

```diff
@@ -56,7 +57,7 @@ const Barrier = ({
                     header={barrier_title}
                     is_center
                 >
-                    <div>
+                    <div onClick={toggleBarriersTable}>
```

Comment:
This function on `div` is temporary solution as we are going to replace `Barrier` component